From e860cd747fb03d40ec6fb6ec79949eb727b1b3ab Mon Sep 17 00:00:00 2001
From: Tri Vo <tri.vo.xb@rvc.renesas.com>
Date: Mon, 14 Nov 2016 19:41:33 +0700
Subject: [PATCH 1308/2066] thermal: Kconfig: Add SOC_BUS dependency for
 RCAR_GEN3_THERMAL

R-Car Gen3 thermal driver now uses new way to detect SoC revision.
This method depends on SOC_BUS so it should be added as dependency
for the driver.
By this way, link error can be avoided when kbuild is executed.

Signed-off-by: Tri Vo <tri.vo.xb@rvc.renesas.com>
[zou:Original patch taken from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git
 v4.9/rcar-3.5.1]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 drivers/thermal/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/thermal/Kconfig b/drivers/thermal/Kconfig
index 5f28172..2f91084 100644
--- a/drivers/thermal/Kconfig
+++ b/drivers/thermal/Kconfig
@@ -176,6 +176,7 @@ config RCAR_THERMAL
 config RCAR_GEN3_THERMAL
 	tristate "Renesas R-Car Gen3 thermal driver"
 	depends on ARCH_RENESAS || COMPILE_TEST
+	select SOC_BUS
 	depends on HAS_IOMEM
 	help
 	  Enable this to plug the R-Car Gen3 thermal sensor driver into the Linux
-- 
1.7.5.4

